.clang-tidy

```diff
@@ -208,6 +208,7 @@ Checks: "bugprone-*,\
          -clang-analyzer-cplusplus.NewDelete,\
          -clang-analyzer-cplusplus.NewDeleteLeaks,\
          -clang-analyzer-optin.performance.Padding,\
+         -clang-analyzer-security.insecureAPI.DeprecatedOrUnsafeBufferHandling,\
```

Comment:
why has this been added?

Reply:
It is a warning that was previously implicitly disabled because it's a C11+-only warning, it's bogus in that it suggests that calls that are safe are unsafe (all the `scanf`s it complains about have a format string that specifies no strings), and it's bogus in that the suggested replacement (`scanf_s`) is optional in C11 and not widely implemented (it's basically a Microsoft-only function).